src/modeler/AutoRest.Swagger/Validation/TrackedResourceValidation.cs

```diff
@@ -138,7 +138,7 @@ private bool HandleSchema(Schema schema, Dictionary<string, Schema> definitions)
             return true;
         }
 
-        private bool listByXCheck(List<Operation> getOperations, Regex regEx, string definitionKey, Dictionary<string, Schema> definitions)
+        private bool listByXCheck(IEnumerable<Operation> getOperations, Regex regEx, string definitionKey, Dictionary<string, Schema> definitions)
```

Comment:
I don't mind making these changes but then I would have to update the whole file (and its dependencies if any). Given that it would be beyond the scope of this PR is that a good idea? Just wondering....